packages/structures/src/channels/ForumTag.ts

```diff
@@ -0,0 +1,57 @@
+import type { APIGuildForumTag } from 'discord-api-types/v10';
+import { Structure } from '../Structure.js';
+import { kData } from '../utils/symbols.js';
+import type { Partialize } from '../utils/types.js';
+
+/**
+ * Represents metadata of a thread channel on Discord.
+ *
+ * @typeParam Omitted - Specify the properties that will not be stored in the raw data field as a union, implement via `DataTemplate`
+ */
+export class ForumTag<Omitted extends keyof APIGuildForumTag | '' = ''> extends Structure<APIGuildForumTag, Omitted> {
+	public constructor(data: Partialize<APIGuildForumTag, Omitted>) {
+		super(data);
+	}
+
+	/**
+	 * The id of the tag.
+	 */
+	public get id() {
+		return this[kData].id;
+	}
+
+	/**
+	 * The name of the tag.
+	 */
+	public get name() {
+		return this[kData].name;
+	}
+
+	/**
+	 * Whether this tag can only be added to or removed from threads by a member with the {@link discord-api-types/v10#PermissionFlagsBits.ManageThreads} permission.
```

Comment:
How come this is `{@link discord-api-types/v10#PermissionFlagsBits.ManageThreads}` but line 83 in packages/structures/src/channels/ThreadMetadata.ts has `{@link PermissionFlagsBits.ManageThreads}`? I assume one of them is wrong.